### DISCUSSION TITLE
Accent colors for color theme (multi-colored workstreams)

### DISCUSSION THREAD

**Discussion by bass4aj** [csgoh/roadmapper#12] [Ideas]
It would be nice to be able to add accent colors (like from a company branding page) to a colourtheme, then use that to apply different colors for each workstream.

**Comment by bass4aj** [comment]
Alternatively, it would be good to at least add methods to set the color directly for a given group, task, or milestone object

**Reply by csgoh** [reply]
Hi, thanks for your suggestion. Changing colour, is currently supported. Please refer to gallery for the sample roadmap and code:  https://github.com/csgoh/roadmapper/wiki/Gallery

**Comment by felix-seifert** [comment]
I like the functionality that users can specify the colours very accurately. This gives them the freedom to adapt roadmapper's design to their needs. It is, however, a quite complex process of defining the colour for many different items. 

It might be interesting to somehow integrate the idea of @bass4aj by wrapping several colour change methods to allow users to create a roadmapper design easily. Say, a user just specifies an primary colour and a secondary colour, and roadmapper will be displayed in a nice design with these two colours.

**Reply by csgoh** [reply]
Thanks for the idea! I would need some thinking on this.

**Comment by ashugaur** [comment]
Excellent module! and thanks for making it. Is it possible to include option to have the Roadmap class include fill_colour="#FFFFFF" or 'transparent'. Currently things show up as white and does not work well with dark mode.

**Reply by csgoh** [reply]
Hi @ashugaur, I will add this feature to next release. Stay tuned! 😊

**Reply by csgoh** [reply]
This has been addressed in v1.3.0

**Reply by ashugaur** [reply]
Great! Works like a charm. [For reference](https://github.com/csgoh/roadmapper/wiki/Usage-Documentation#step-3-set-the-background-colour-this-step-is-optional).
